frontend/src/helper/Validation-helper.ts

```diff
@@ -0,0 +1,74 @@
+import {validateErrors} from "./Error-helper.ts";
+import {CSSProperties} from "react";
+
+export const validationDanger = "text-danger"
+export const validationWarning = "text-warning"
+export const validationSuccess ="text-success"
+
+export type ValidationMessageColors = typeof validationDanger | typeof validationWarning | typeof validationSuccess
+
+export const validationTextSmall : CSSProperties = {fontSize: 14}
+export const validationTextBig : CSSProperties = {fontSize:16}
+
+export type ValidationMessageSize = typeof validationTextSmall | typeof validationTextBig
+
+export type Validation={
+    required?: {
+        value:boolean,
+        message: string,
+    },
+    minLength?: {
+        value:number
+        message: string
+    },
+    maxLength?: {
+        value:number,
+        message:string
+    },
+    pattern?: {
+        value: RegExp,
+        message: string,
+    }
+}
+
+export const name_validation : Validation = {
+        required: {
+            value: true,
+            message: 'Username is required.',
+        },
+        minLength: {
+            value:3,
+            message: 'Username must be at least 3 characters long'
+        },
+        maxLength: {
+            value: 20,
+            message: 'Username can\'t exceed 30 characters.',
+        },
+}
+
+export const password_validation = {
```

Comment:
Doesnt match Backend validation.

` @ Size(min = 8, max = 100)

@ Pattern(regexp = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d).*$", message = "Password must contain at least one uppercase letter, one lowercase letter, and one number.")`

Reply:
Ah save darauf habe ich gar nicht geachtet 🤣